### PR TITLE
Added Indian Helpline to the list.

### PR DESCRIPTION
**Pull request by hemanth**
@conorfl 

------
IA Page:  http://duck.co/ia/view/help_line

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/help_line/helplines.yml [hunk 1]
```diff
@@ -49,3 +49,8 @@ US:
   - phone: 1-800-273-TALK (8255)
     web: http://www.suicidepreventionlifeline.org/
   display_country: the U.S.
+IN:
```
Comment (moollaza) [inline share/goodie/help_line/helplines.yml]: Mind adding this below `IE`. The list is alpha sorted

**Comment by moollaza** [comment]
@hemanth please be sure to also provide an IA page link in your PR description!

That allows us to deploy PR's to Beta for testing (this is a case that likely doesn't need a beta deploy, but it's good practice!)

**Comment by hemanth** [comment]
@moollaza Cool, done.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Comment by hemanth** [comment]
Thanks.

**Pull request merged**